moped-database/migrations/1654646507541_proj_list_view_fund_src_filter/up.sql

```diff
@@ -39,7 +39,7 @@ CREATE OR REPLACE VIEW "public"."project_list_view" AS
     mp.capitally_funded,
     mp.date_added,
     mp.added_by,
-    mp.is_deleted,
+    mp.is_retired,
```

Comment:
I needed to return this migration that was edited back to the state in which it was applied to staging. This migration never applied when merged to `main` so the `project_list_view` did not agree with the permissions metadata in `tables.yaml` that contains `is_deleted` instead of `is_retired`. 

The migration below will make the change so the state of `project_list_view` will agree with the tables permissions metadata.